.github/workflows/tox.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Tox
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Run tox tests
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.6'
+          - '3.7'
+          - '3.8'
+    runs-on: ubuntu-18.04
```

Comment:
20.04 is going to be the default shortly (ubuntu-latest). Any particular reason to pin to 18.04?

Reply:
This is mostly because we have precompiled Ice wheels for 3.6/3.7/3.8 for this environement used in `tox.ini`. Happy to switch if we have equivalent artifacts for Ubuntu 20.04 and the three Python versions

Reply:
3.9 should also exist now ;)

Reply:
I am fine with Python 3.6/3.7/3.8/3.9 on Ubuntu 20.04 too. Should all the wheels be generated as release artifacts of https://github.com/ome/zeroc-ice-ubuntu2004 then?

Reply:
miniconda is installed by default, we could skip these custom built wheels and use that instead?

Reply:
For reference, https://github.com/sbesson/omero-marshal/actions/runs/393725465 is the result of the execution on Ubuntu 20.04, with Python 3.9 and without the custom wheels. The run time increased from ~1min to ~10min largely because of the installation of `zeroc-ice` but if we all agree this is acceptable, I can push the change to this PR